Automattic/wp-calypso
# Stats: Set quantity value based on period

Related to #83039

## Proposed Changes

Determines the quantity value (used when fetching stats) based on the provided period (in the URL).

Follows #83097 which should be merged first.

## Testing Instructions



1. Open the Live branch.
2. Navigate to the **Stats → Traffic** page.
3. Enable the new date picker: `?flags=stats/date-control`
4. Choose the "Last Year" shortcut.
5. Confirm that only 13 columns are drawn in the chart.
6. Choose the "Last 30 Days" shortcut.
7. Confirm chart updates correctly.
8. Open date picker and manually change the start date to July 1st.
9. Confirm the chart updates correctly showing enough weeks to cover the partial range. (ie: July 1st is not a Monday so we would lose two days without the extra slice).

## Pre-merge Checklist
